src/client/src/services/edgeService.ts

```diff
@@ -16,10 +15,11 @@ const sendEdge = async (edge: IEdge): Promise<boolean> => {
 };
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const deleteEdge = async (edge: Edge<IEdge>): Promise<void> => {
-    await axiosWrapper(
-        axios.delete(`${baseUrl}/${edge.source}/${edge.target}`)
-    );
+const deleteEdge = async (
+    source: string | number,
+    target: string | number
```

Comment:
If the typing is fixed, why do are both string and number the possible types of the source/target?